pkg/cmd/cli/describe/printer.go

```diff
@@ -161,6 +161,21 @@ func PrintTemplateParameters(params []templateapi.Parameter, output io.Writer) e
 	return nil
 }
 
+// Receives a resource kind and its name, returns a formatted
+// string with the "resource_kind/resource_name"
+func formatResourceName(kind, name string, withKind bool) string {
+	if !withKind {
+		return name
+	}
+
+	slash := "/"
+	if kind == "" {
+		slash = ""
+	}
+
+	return (kind + slash + name)
```

Comment:
The parentheses here are unnecessary, and not common use in Go (see for example the `if` and `for` statements).